doc/operators_precedence.rst

```diff
@@ -0,0 +1,56 @@
+Unary operators precedence:
```

Comment:
This doc is the most important part of this PR.
We need to review the precedence carefully.